scene/main/canvas_layer.cpp

```diff
@@ -241,8 +272,11 @@ void CanvasLayer::_bind_methods() {
 	ObjectTypeDB::bind_method(_MD("set_scale","scale"),&CanvasLayer::set_scale);
 	ObjectTypeDB::bind_method(_MD("get_scale"),&CanvasLayer::get_scale);
 
+	ObjectTypeDB::bind_method(_MD("set_custom_viewport","viewport:Viewport"),&CanvasLayer::set_custom_viewport);
+	ObjectTypeDB::bind_method(_MD("get_custom_viewport:Viewport"),&CanvasLayer::get_custom_viewport);
+
 	ObjectTypeDB::bind_method(_MD("get_world_2d:World2D"),&CanvasLayer::get_world_2d);
-	ObjectTypeDB::bind_method(_MD("get_viewport"),&CanvasLayer::get_viewport);
+//	ObjectTypeDB::bind_method(_MD("get_viewport"),&CanvasLayer::get_viewport);
```

Comment:
That's a bug according to @reduz in https://github.com/godotengine/godot/issues/5696#issuecomment-237275585 :
> @bojidar-bg
> Many nodes override inherited methods, which makes the said methods uncallable from GDScript.
> Here is a small list
>
> Viewport::get_viewport
> CanvasItem::get_viewport
> EditorPlugin::get_name

> @reduz 
> ah, that is clearly a bug